buildsystem/generator/make.go

```diff
@@ -0,0 +1,300 @@
+package generator
+
+import (
+	"io/ioutil"
+
+	"github.com/evergreen-ci/shrub"
+	"github.com/evergreen-ci/utility"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
```

Comment:
I suspect you can just use ReadYAML/WriteYAML from utility.

Reply:
I would prefer to `yaml.UnmarshalStrict` rather than just `yaml.Unmarshal`, which is what utility does.